service/client.go

```diff
@@ -72,4 +72,14 @@ type Client interface {
 
 	// Returns whether we attached to a running process or not
 	AttachedToExistingProcess() bool
+
+	// Returns the address of a location expression
+	// loc ::= <filename>:<line> | <function>[:<line>] | /<regex>/
+	// * <filename> can be the full path of a file or just a suffix
+	// * <function> ::= <package>.<receiver type>.<name> | <package>.(*<receiver type>).<name> | <receiver type>.<name> | <package>.<name> | (*<receiver type>).<name> | <name>
+	// * <function> must be unambiguous
+	// * /<regex>/ will return a breakpoint for each function matched by regex
+	// * *<address> sets a breakpoint at a program address
+	// NOTE: this function does not actually set breakpoints.
+	FindLocation(loc string) ([]api.Location, error)
```

Comment:
This should also accept something like `break <line>` to set a breakpoint at <line> of current file.